.editorconfig

```diff
@@ -15,5 +15,4 @@ charset = utf-8
 end_of_line = crlf
 
 [*.yml]
-indent_style = space
```

Comment:
Why this change? YAML files are space indented.

Reply:
Your ide will tell u: not necessary here.

Reply:
The `editorconfig` file is meant to tell the formatting style to IDE.

Reply:
The proper justification for removing this is it's redundant since we already have `indent_style = space` under `[*]`.